docs/src/api/class-route.md

```diff
@@ -503,6 +503,12 @@ If set changes the request URL. New URL must have same protocol as original one.
 Maximum number of request redirects that will be followed automatically. An error will be thrown if the number is exceeded.
 Defaults to `20`. Pass `0` to not follow redirects.
 
+### option: Route.fetch.timeout
+* since: v1.33
```

Comment:
I see the issue was tagged `1.34`, I'll update this?

Reply:
No, it is right on time for 1.33 for now. If anything, I'll bump to 1.34 before the release.